Makefile

```diff
@@ -65,12 +65,11 @@ coverage: ## check code coverage quickly with the default Python
 		$(BROWSER) htmlcov/index.html
 
 docs: ## generate Sphinx HTML documentation, including API docs
-	rm -f docs/bigchaindb.rst
-	rm -f docs/modules.rst
-	sphinx-apidoc -o docs/ bigchaindb
```

Comment:
Turns out `sphinx-apidoc` is a way to autogenerate docstrings references (e.g. `automodule`s and the like).

We don't use this currently, but it might be nice to turn on in the future.